x-pack/solutions/observability/plugins/observability/public/pages/alert_details/components/related_dashboards/dashboard_tile.tsx

```diff
@@ -35,20 +32,28 @@ export function DashboardTile({
   dashboard,
   actionButtonProps,
 }: {
-  dashboard: DashboardMetadata;
+  dashboard: RelatedDashboard;
   actionButtonProps?: ActionButtonProps;
 }) {
   const {
     services: {
       share: { url: urlService },
+      savedObjectsTagging: { ui: savedObjectsTaggingUi },
     },
   } = useKibana();
   const dashboardLocator = urlService.locators.get<DashboardLocatorParams>(DASHBOARD_APP_LOCATOR);
 
+  const tagsReferences: SavedObjectsReference[] = dashboard.tags?.length
+    ? dashboard.tags.reduce((acc, value) => {
+        const reference = savedObjectsTaggingUi.convertNameToReference(value);
```

Comment:
What if `convertNameToReference` throws? I suggest you wrap it in a try/catch and cover it with some tests